src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+    private String color = getColor();
+
+    public Square(int side, String color) {
+        this.side = side;
+        this.color = color;
+    }
+
+    @Override
+    public double areaCalculator() {
+        return (side * side);
+    }
```

Comment:
The method name 'areaCalculator' should not be a verb. Consider renaming it to 'calculateArea' to follow Java naming conventions.